docs/index.rst

```diff
@@ -733,25 +733,27 @@ solution, Confluent Replicator is also configured with security.
 
    .. figure:: images/replicator_topic_info.png
 
+5. Notice that because |crep| default is ``topic.config.sync=true`` (see |crep| :ref:`documentation <rep-destination-topics>`), then the replicated
+   topic ``wikipedia.parsed.replica`` has enabled |sv| just like the original
+   topic ``wikipedia.parsed``.
```

Comment:
```suggestion
   topic ``wikipedia.parsed`` (you may need to click on the ``Show full config`` button to see all values).
```

Reply:
Great suggestion @rspurgeon .  I modified it slightly but added the gist.